starknet-core/src/types/block.rs

```diff
@@ -80,10 +86,22 @@ mod tests {
 
         let block: Block = serde_json::from_str(raw).unwrap();
 
-        assert_eq!(block.block_number, 47543);
+        assert_eq!(block.block_number.unwrap(), 47543);
         assert_eq!(block.transaction_receipts.len(), 4);
         let receipt = &block.transaction_receipts[3];
         assert_eq!(receipt.events.len(), 1);
         assert_eq!(receipt.events[0].data.len(), 2);
     }
+
+    #[test]
+    fn test_block_deser_pending() {
+        // has an L2 to L1 message
```

Comment:
The comment doesn't make sense in this context, probably a copy pasta err?

Reply:
Yes copy pasta got caught